packages/cli/src/create/helpers/create-contract.ts

```diff
@@ -46,8 +46,13 @@ export async function createContractProject({
 }: ICreateContractProject) {
   // Check that the selected base contract is valid
   if (baseContract) {
-    const found = hasBaseContract(baseContract);
-
+    let found = false;
+    if (baseContract.includes(",")) {
+      const [account, factory] = baseContract.split(",");
```

Comment:
why do we need to split here?

Reply:
I imagine in the CLI you select "smart wallet" and it just generates 2 contracts for you right?

Reply:
yeah exactly - can remove and just create an Account contract (would be cleaner) but at the time thought it would be better for users to have the factory contract for free